test/fuzz/test_runner.py

```diff
@@ -380,7 +409,7 @@ def job(t, args):
             done_stat = [l for l in output.splitlines() if "DONE" in l]
             assert len(done_stat) == 1
             stats.append((target, done_stat[0]))
-
+            remaining_targets = log_fuzz_progress(remaining_targets, test_list, done_stat, target)
     if using_libfuzzer:
         print("Summary:")
```

Comment:
The summary at the end should probably be removed now?

Reply:
I just made into a debug log in [c54df63](https://github.com/bitcoin/bitcoin/pull/32354/commits/c54df63f7d5047bf8efc345f102942cbb86fb019)